package.json

```diff
@@ -59,7 +58,7 @@
     "@patternfly/react-table": "5.3.3",
     "@patternfly/react-tokens": "5.3.1",
     "date-fns": "3.6.0",
-    "deep-equal": "2.2.3",
+    "dequal": "2.0.3",
```

Comment:
Got it.

I think maybe we need to split up hooks.ts then?  Or avoid using it entirely.

But I might also imagine that dequal isn't going to bring a lot of dependabot spam, either.  Last update was two years ago, and the one before was 4.

@martinpitt also proposed just copying the code...

Reply:
btw: hooks.ts gets checked not because everything in pkg/lib/ gets checked.  We only include src/* in the compilation.  But anything that gets imported is dragged in.

There's some module isolation features in TS designed to avoid issues like this in large monorepos.  Maybe they help?

Reply:
> I think maybe we need to split up hooks.ts then? Or avoid using it entirely.

That feels going backwards, we want to re-use code.

> But I might also imagine that dequal isn't going to bring a lot of dependabot spam, either. Last update was two years ago, and the one before was 4.
> @martinpitt also proposed just copying the code...

We already had this discussion in the cockpit PR and decided to go with inclusion as `pkg/lib` already includes external deps like `xterm`. 

> btw: hooks.ts gets checked not because everything in pkg/lib/ gets checked. We only include src/* in the compilation. But anything that gets imported is dragged in.

I know, I already fixed the issue where it scanned `pkg/lib` in files in `test/static-code` for no reason. Sadly because it gets imported its still an issue.

> There's some module isolation features in TS designed to avoid issues like this in large monorepos. Maybe they help?

Honestly speaking, we have better things to do then argue over this.

Reply:
I was going to approve this but obviously this is not a `dependency` but a `devDepenedency`